src/ImageUploader.tsx

```diff
@@ -29,7 +31,9 @@ const ImageUploader: React.FC<ImageUploaderProps> = ({ onClose }) => {
     const handleUpload = () => {
         if (fileList) {
             console.log('Uploading images:', fileList);
-            alert('Images uploaded successfully!');
+            {
+                /*alert('Images uploaded successfully!');*/
+            }
```

Comment:
Let's look into using the api here since the back end integration has started. Have a look [here](https://github.com/ias-tcd/competitor-matching-web-app/pull/28/commits/a573089c8fe9815f3b5a23e9ed5ccac46ae83076#diff-a721ccf997cc31090cf5a1781fa8a54cb75492e6ab21d6eff113f06742c9fff1R34-R44) at this example on how we might send the request